addons/mail/static/src/messaging/component/activity/activity_tests.js

```diff
@@ -384,11 +384,11 @@ QUnit.test('activity details toggle', async function (assert) {
     const tomorrow = new Date();
     tomorrow.setDate(today.getDate() + 1);
     const activity = this.env.entities.Activity.create({
-        activity_type_id: [1, 'Fake type'],
-        create_date: date_to_str(today),
-        create_uid: [1, 'Admin'],
-        date_deadline: date_to_str(tomorrow),
+        creator: [['insert', { id: 1, _displayName: 'Admin' }]],
```

Comment:
Not a typo, initially `nameOrDisplayName` was `displayName`, so it was to distinguish both "fields".

Ideally this should be `displayName` that is a related on partner, so that source of this data is on related partner. Problem with that is that we may be aware of a user but not of related partner (or vice-versa), so this is field acts as a placeholder for displayed name of user without a relation to partner yet.

All of this shows an issue that I just decided to postponed to a later moment:

- enabling `update()` on related fields...
- ...that doesn't exist...
- ...and enforcing user always being linked to a partner...
- ...that we don't know its id yet, so define a "placeholder" partner...
- ...that may be replaced to another partner that we know at a later date that it corresponds to partner of this user (all of this includes changing direct relations of "placeholder" partner)

Maybe all of this looks harder than it really is, but I didn't feel confident enough to solve that problem just yet. Fortunately only entity `User` has this problem for now!

Reply:
I don't think the "user" of the entity should ever have to pass an _ field. If it is related to partner, let's just call it `partnerDisplayName`?
I think I understand the logic here, I'm just commenting about the name itself that looks private but is passed from outside which feels wrong.